migrator.sh

```diff
@@ -398,7 +423,7 @@ query_source_images() {
     for i in ${REPO_LIST}
     do
       # get list of tags for image i
-      IMAGE_TAGS=$(curl ${V1_OPTIONS} -sf ${V1_PROTO}://${AUTH_CREDS}@${V1_REGISTRY}/v1/repositories/${i}/tags | jq -r 'keys | .[]') || catch_error "curl => API failure"
+      IMAGE_TAGS=$(curl ${V1_OPTIONS} -sf ${V1_PROTO}://${AUTH_CREDS}@${V1_REGISTRY}/v1/repositories/${i}/tags | jq -r 'keys | .[]') || skipping "curl => API failure" "${i}"
```

Comment:
Ah yes, I see you've made a commit to your `master` branch.  I'd create a feature branch from your master and then revert your last commit once you've created and checked out the new feature branch.

```
git branch <my descriptive name>
git reset --hard HEAD~1
```

You should be able to see in `git log` that it only includes the first commit then.

Then you can push that branch and you'll need to submit a new PR for the new feature branch.